### DISCUSSION TITLE
Install the driver from the terminal

### DISCUSSION THREAD

**Discussion by hamedprog** [VirtualDrivers/Virtual-Display-Driver#177] [Q&A] [answered]
I used pnputil to install the driver. The output shows the driver was added successfully, but I can't see it in Device Manager. i need to install driver using terminal. please help

**Comment by bud3699** [comment]
You cannot use pnputil alone to install the driver, that wont install the device as well

**Reply by hamedprog** [reply]
Thanks for the reply. I have many VMs and I want to install this driver on all of them. Is there no automated way to install it?

**Reply by bud3699** [reply, marked as answer]
I do have a script which works on installing the beta. I was holding off on releasing it until we get a next full release out (Hopefully soon)

**Reply by hamedprog** [reply]
Thank you for the update. I appreciate your efforts in creating the script and your consideration in timing the release. Looking forward to the full release and the enhancements it will bring. Thanks again for your dedication to this project!
